python/pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "aac-spec"
-version = "0.1.4"
+version = "0.1.5"
 authors = [
 	{ name="bunchw", email="asdfasdaf@email.com" },
 	{ name="lizzcondrey", email="asdfasdfd@mail.com" },
```

Comment:
And looks like things are working fine with moving to the team author reference, this got missed for being changed it looks like.